src/style.css

```
@@ -1 +1,30 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 > :nth-child(0),
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
```

let's make all selectors consistent.
> is redundant, and no need to set active star for stars--0

```suggestion
.stars--1  :nth-child(-n + 1),
.stars--2  :nth-child(-n + 2),
.stars--3  :nth-child(-n + 3),
.stars--4  :nth-child(-n + 4),
.stars--5  :nth-child(-n + 5) {
  background-image: url('./images/star-active.svg');
}
```